server/etcdserver/api/membership/membership_test.go

```diff
@@ -34,10 +34,6 @@ func TestAddRemoveMember(t *testing.T) {
 	c.AddMember(newTestMember(18, nil, "node18", nil), true)
 	c.RemoveMember(18, true)
 
-	// Skipping removal of already removed member
```

Comment:
Looks like this test was totally incorrectly written?! It dependent on fact that v2store was not set at all.